apis/Google.Cloud.Metadata.V1/Google.Cloud.Metadata.V1.IntegrationTests/project.json

```diff
@@ -25,6 +24,7 @@
           "type": "platform"
         }
       }
-    }
+    },
+    "net451": {}
```

Comment:
Okay, that's bizarre - but may explain the issues I've been having. (I've been just running from the command line for a while.)

If the rest of the change were larger, I'd suggest splitting the commit, but I don't think it's worth it.